compiler/mir/mirenv.nim

```diff
@@ -43,8 +43,13 @@ type
       ## includes both normal globals and threadvars
     procedures*: SymbolTable[ProcedureId, PSym]
 
+    numbers*: BiTable[BiggestInt]
```

Comment:
Would it be too much to have an `mnkEnumField` kind and a store for the field values?

Reply:
I think it is, yeah. The existence of enum fields is irrelevant to the mid-end and code generators, as they're all already lowered into just integer values.

Nonetheless, if you want, I can update the pretty-printer to render enum values as their field names again.

Reply:
> Nonetheless, if you want, I can update the pretty-printer to render enum values as their field names again.

That would be nice if not too much trouble, just makes tracing and debugging easier.

Reply:
I've updated the MIR pretty-printing of literals to produce results that match the previous ones (the PR description is updated already).

Reply:
Thanks